guides/IMPORT_EXISTING_DOCUMENTS.md

```diff
@@ -0,0 +1,67 @@
+The `firestore-meilisearch` script is for use with the official Firebase Extension [**Meilisearch**](https://github.com/firebase/extensions/tree/main/firestore-meilisearch).
+
+### Overview
+
+The import script can read all existing documents in a Cloud Firestore collection or in sub-collections and index them into an index in Meilisearch.
+
+#### Important notes
+
+- You must run the import script over the entire collection **_after_** installing the Meilisearch extension; otherwise the writes to your database during the import might not be exported to the your `${param:MEILISEARCH_INDEX_NAME}` index.
+
+- The import script may take up to a while to complete. If your collection is large, you may want to consider importing in batches.
+
+- You cannot use wildcard notation in the collection path (i.e. /collection/{document}/sub_collection}). Instead, you can use a collectionGroup query. To use a collectionGroup query, provide the collection name value as ${COLLECTION_PATH}, and set ${COLLECTION_GROUP_QUERY} to true.
+
+Warning: A collectionGroup query will target every collection in your Firestore project with the provided ${COLLECTION_PATH}. For example, if you have 10,000 documents with a sub-collection named: actors, the import script will query every document in 10,000 actors collections.
+
+### Run the script
+
+The import script uses several values from your installation of the extension:
```

Comment:
`from the configuration provided when you installed the extension` ?

Reply:
Yes, the configuration you choose when you install the extension will be needed by the script too. Like the `id` of your Firebase project.

Reply:
```suggestion
The import script uses several values from the configuration provided when you installed the extension:
```